### PR TITLE
Setup "Hello Microverse" project file

### PR DESCRIPTION
**Pull request by Errytagedesign**
In this branch I did:

 - Add index.html file.
 - Add style.css file.
 - Updated Readme.md
 - Add HTML and CSS Linters

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,181 @@
...
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+<a href="https://codeeq-verse.netlify.app/">Live Demo Link</a>
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
```
Comment (zieeco) [inline README.md:66]: - Hey, your live demo link is currently broken. please, kindly update it to be working or kindly remove this live demo section from the readme body and from the top navigation on the readme to make it more professional.

File: README.md [hunk 2]
```diff
@@ -1 +1,181 @@
...
+- in order to run the linter test on stylesheets:
+
+```sh
+ npx styleint "**/*.{css,scss}"
+```
+
+### Deployment
+
+- You can easily use github pages to deploy your project
+  <a href="https://docs.github.com/en/pages/getting-started-with-github-pages/about-github-pagess">Useful link about Github Pages</a>
```
Comment (zieeco) [inline README.md]: - Hey, let's also update this link as the link is broken. You can tell users the platforms they can use to deploy the app, e.g.: Github pages, netlify, etc. Let's update this to make the readme more professional.

**Review by zieeco — CHANGES_REQUESTED** [review]
Hi @Errytagedesign,





Great job so far! 👏 
You've done well, however, there are some issues that you still need to work on to go to the next project, but you are almost there!


## To highlight 🎯

- Great project setup.✔️
- Added linter correctly. ✔️
- Descriptive PR. ✔️


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
- N/A


**_Every comment with the **[OPTIONAL]** prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._**


Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 1 above)
Comment (Errytagedesign) [reply README.md:66]: This is fixed, kindly check again.
Thank you.

File: README.md (hunk 2 above)
Comment (Errytagedesign) [reply README.md]: This is also updated with working links.

Comment (Errytagedesign) [reply README.md]: @zieeco Do i need to re-submit this PR link for re-review or not?

**Review by skyv26 — APPROVED** [review]
## Hi @Errytagedesign, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit:  
Congratulations! 🎉 

## STATUS: APPROVED 🟢 



## Highlights
- Professional `README` template ✅ 
- Correct `GitHub` flow ✅ 
- Well Implemented the feature ✅ 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @skyv26 in your question so I can receive the notification. 

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**